code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -388,6 +388,21 @@
 	new /obj/item/weapon/card/id/prisoner/random(src)
 	new /obj/item/device/radio/headset(src)
 
+/obj/structure/closet/secure_closet/vr
+	name = "VR exile"
+	req_access = list(access_security)
+	anchored = 1
+
+/obj/structure/closet/secure_closet/vr/New()
+		..()
+		new /obj/item/clothing/ears/vr_headset/exile(src)

```

Comment:
You could simplify this with a for loop.

Reply:
This is how all existing closets was done, so I did not want to fix what was not broken.

Reply:
Well it's not fixing it, both ways work it's just I find for loops more readable especially with a large number of items. You can simplify this to 2 lines of code.

https://github.com/ParadiseSS13/Paradise/blob/3ed55007d47bb5510825747a8dd7cfff2691e26e/code/game/objects/structures/crates_lockers/closets/secure/medical.dm#L144